README.md

```diff
@@ -1,37 +1,155 @@
+* `Glib-2.0` for supporting C functions.
+
+If you are using a distribution of the Debian family (including Ubuntu) you can get all of these dependencies by opening a Terminal window and typing:
+
+```
+sudo apt install libglib2.0-dev libhamlib-dev libncurses5-dev libtinfo-dev libxmlrpc-core-c3-dev
+```
+
+While not strict build dependencies, the following packages may be helpful too:
+
+* `sox` for audio signal handling during SSB contests and
+* `xplanet` which allows you to see the latest DX spots on the globe.
+
+On Debian Linuxes:
+
+```
+sudo apt install sox xplanet
+```
+
+Once the dependencies are installed, the easiest way to install tlf is by downloading the latest tarball (version 1.4.1) from [here](http://download.savannah.gnu.org/releases/tlf/tlf-1.4.1.tar.gz), then navigating your Terminal to the directory where you unpacked it, and typing:
+
+```
+./configure
+make
+sudo make install
+```
+
+If you plan to do contests in digimodes using Fldigi, start the sequence above with
+
+```
+./configure --enable-fldigi-xmlrpc
+```
+
+followed by the `make` and `make install` commands.
+
+If you are compiling tlf from a clone of the Github repository instead of the tarball, please do
+
+```
+autoreconf --install
+```
+
+before the above commands.
+
+By default, TLF will install into `/usr/local/bin`, together with the
+shell scripts from the scripts directory. Data files will install into
+`/usr/local/share/tlf`.
+
+If you want to change any of these paths, you can do that with the
+configure script too, e.g. 
+
+```
+./configure --prefix=/usr --datadir=/usr/share
+```
+
+will install TLF and scripts into `/usr/bin`, datafiles into `/usr/share/tlf`.
+
+If you want to use [cwdaemon](https://github.com/acerion/cwdaemon) for CW, make sure it's installed, and either set it up to start at system startup, or plan to start it manually before launching TLF. 
+
+If you want to use a K1EL Winkeyer for CW, download and install either [winkeyer_server](https://github.com/ok2cqr/winkeyer_server) from OK2CQR or N0NB's [winkeydaemon](https://github.com/N0NB/winkeydaemon), following the instructions on those pages. You'll need to start the Winkeyer server (either one) before starting TLF. Both programs work by impersonating cwdaemon, which TLF will use automatically if you've enabled the NETKEYER settings in the `logcfg.dat` file.
+
+## Quick start
+
+TLF can be used in all types of contests, but regardless of the mode, much of the initial setup will be the same. This example will assume a USA-based station is setting up for the ARRL DX contest - simply substitute the appropriate rules file for whatever contest you're entering.
+
+The easiest way to keep things organized is to create a new directory for each contest, then copy the default configuration files into it. You can do this in your file manager, or at the command line:
+
+```
+mkdir arrldx
+
+cd arrldx
+
+mkdir rules
+
+cp /usr/local/share/tlf/logcfg.dat .
```

Comment:
This assumes an installation to /usr/local. User may have a stock version. (see Installation)

Reply:
Where would it be in the stock installation?

Reply:
For Debian it's under `/usr/share/tlf/` (so just /usr without /local)